packages/flutter/test/material/tabs_test.dart

```diff
@@ -2677,6 +2691,142 @@ void main() {
     // There was a time where this would throw an exception
     // because we tried to send a notification on dispose.
   });
+
+  testWidgets('TabController animation should not get interrupted when user drags TabBarView', (WidgetTester tester) async {
```

Comment:
can you also add a test to make sure tap a tab can not interrupt the animation?

Reply:
also it will be perfect if you can add another test about the clipping bug you fixed as part of this pr

Reply:
Done!